src/util.ts

```diff
@@ -864,9 +864,9 @@ export async function scheduleAsyncTask<T>(task: () => Promise<T>): Promise<T> {
     });
 }
 
-export function isFileInsideFolder(openEditor: vscode.TextDocument, folderPath: string): boolean {
+export function isFileInsideFolder(openEditor: vscode.Uri, folderPath: string): boolean {
```

Comment:
I don't think this needs to change. Please revert.

Reply:
Oh, I see that the rename event is the cause for this signature change.  I think that should have been split out from this PR.

Reply:
The rename event only checks for existance of CMakeLists.txt. That's why I added it to this PR.

```ts
            if (filePath.endsWith("cmakelists.txt")) {
                await this.doCMakeFileChangeReconfigure(file.newUri);
            }
```

Reply:
I changed the title of the PR